<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MakeStep</class>
 <widget class="QWidget" name="MakeStep">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>428</width>
    <height>384</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="page_1">
      <layout class="QFormLayout" name="formLayout_2">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::ExpandingFieldsGrow</enum>
       </property>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="makeLabel">
         <property name="text">
          <string>Override %1:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="makeLineEdit"/>
       </item>
       <item>
        <widget class="QLabel" name="makeArgumentsLabel">
         <property name="text">
          <string>Make arguments:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="makeArgumentsLineEdit"/>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>255</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>makeLineEdit</tabstop>
  <tabstop>makeArgumentsLineEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
